metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java

```diff
@@ -262,6 +253,23 @@ public void rename(
         }
     }
 
+    @Override
+    public boolean exists(@Nonnull final ConnectorContext context, @Nonnull final QualifiedName name) {
+        boolean result = false;
+        try (Connection connection = this.dataSource.getConnection()) {
+            final String databaseName = name.getDatabaseName();
+            connection.setSchema(databaseName);
+            final DatabaseMetaData metaData = connection.getMetaData();
+            final ResultSet rs = metaData.getTables(databaseName, databaseName, name.getTableName(), TABLE_TYPE);
+            if (rs.next()) {
```

Comment:
Using next() should be acceptable with the assumption that databases will not have two tables with the same name.